gtfu/src/main/java/gtfu/Util.java

```diff
@@ -969,4 +971,14 @@ public boolean verify(String arg0, SSLSession arg1) {
         HttpsURLConnection.setDefaultHostnameVerifier(validHosts);
     }
 
+    public static boolean isValidJSON(String jsonString){
+        try{
+            ObjectMapper objectMapper = new ObjectMapper();
+            JsonNode jsonNode = objectMapper.readTree(jsonString);
+        } catch(JsonProcessingException e){
+            Debug.log("exception: " + e);
```

Comment:
We need something more descriptive here. Chances are that at some point this will show up in a log file, and then no one will know where to look for the root cause. Stack trace is always nice, and I would also include the offending json string

Reply:
Thanks. I updated a bit, and here is sample output:
```* Invalid JSON detected. Error message: com.fasterxml.jackson.core.io.JsonEOFException: Unexpected end-of-input: expected close marker for Array (start marker at [Source: (StringReader); line: 1, column: 1])
 at [Source: (StringReader); line: 1074, column: 1]```

I think printing the whole list would be overkill, especially since with some agencies it is a huuuge file.

Reply:
Sorry, I'm not seeing the update. Also, I meant a stack trace to show who called this routine. The details for the json are good, but we want the trace as well.